src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -24,6 +24,9 @@ public partial class DllNotFoundException : System.TypeLoadException
         public DllNotFoundException(string message) { }
         public DllNotFoundException(string message, System.Exception inner) { }
     }
+    public partial struct RuntimeArgumentHandle
```

Comment:
RuntimeArgumentHandle is not listed in https://github.com/dotnet/standard/; and it is part of the removed bundle. I do not see why we should have it. @danmosemsft ?